poi-ooxml/src/main/java/org/apache/poi/xwpf/usermodel/XWPFTable.java

```diff
@@ -155,12 +155,12 @@ public XWPFTable(CTTbl table, IBody part, int row, int col) {
         }
     }
 
-    public XWPFTable(CTTbl table, IBody part) {
+    public XWPFTable(CTTbl table, IBody part, boolean initRow) {
```

Comment:
you must keep the original constructor for backward compatibility - that can call this new constructor with initRow=true and you can drastically reduce the size of this PR by using the old constructor unless you want to pass initRow=false